README.md

```diff
@@ -4,6 +4,12 @@ VapourTrail is an interactive bus map made from OpenStreetMap data.
 
 ![demo](img/demo.gif)
 
+VapourTrail aim to be:
+* a large scale or local solution for bus based on OpenStreetMap data
+* an interactive map
+* schematic bus route display
+* an API for bus routes
+
```

Comment:
What would you think of: 
VapourTrail aims to be a large scale or local solution for visualizing bus lines based on OpenStreetMap data composed of:
* an interactive map with a schematic bus route display
* an API for bus routes
* a tileset of vector tiles